test/Microsoft.AspNet.Mvc.TagHelpers.Test/ValidationSummaryTagHelperTest.cs

```diff
@@ -93,7 +93,7 @@ public class ValidationSummaryTagHelperTest
             // Arrange
             var validationSummaryTagHelper = new ValidationSummaryTagHelper
             {
-                ValidationSummaryValue = "ModelOnly"
+                ValidationSummary = ValidationSummary.All,
```

Comment:
this was a small nit: noticed tests did not confirm `excludePropertyErrors` argument to `GenerateValidationSummary()` was `false` in this case